ptf_tests/common/utils/config_file_utils.py

```diff
@@ -199,16 +197,69 @@ def get_gnmi_params_simple(data):
             return None
         for field in mandatory[device_type]:
             param += f"{field}:{port[field]},"
-
+        
         for field in optional:
             if field in port.keys():
                 param += f"{field}:{port[field]},"
-
+   
         param += f"port-type:{port['port-type']}"
-
         params.append(param)
 
     return params
+    
+def get_gnmi_all_port_params_simple(data):

```

Comment:
The function name doesn't convey the purpose. maybe get_gnmi_phy_with_ctrl_port or something similar.

Reply:
updated